### PR TITLE
QF-2067 Manage requests page active

### PR DESCRIPTION
**Pull request by ChrisJWoodcock**

### CODE REVIEW

**Review by karen-hanna — DISMISSED** [review]

File: src/EmployerRequestApprenticeTraining/SFA.DAS.EmployerRequestApprenticeTraining/Application/Queries/GetSettings/GetSettingsResult.cs [hunk 1]
```diff
@@ -0,0 +1,8 @@
+﻿namespace SFA.DAS.EmployerRequestApprenticeTraining.Application.Queries.GetSettings
+{
+    public class GetSettingsResult
+    {
+        public int ExpiryAfterMonths { get; set; }
+        public int RemovedAfterExpiryMonths { get; set; }
+    }
+}
```
Comment (karen-hanna) [inline src/EmployerRequestApprenticeTraining/SFA.DAS.EmployerRequestApprenticeTraining/Application/Queries/GetSettings/GetSettingsResult.cs:8]: Should this also include ProviderRemovedAfterRequestedMonths which is included in the api code?

File: src/Shared/SFA.DAS.SharedOuterApi/InnerApi/Responses/RequestApprenticeTraining/GetSettingsResponse.cs [hunk 2]
```diff
@@ -0,0 +1,11 @@
+﻿using System.Diagnostics.CodeAnalysis;
+
+namespace SFA.DAS.SharedOuterApi.InnerApi.Responses.RequestApprenticeTraining
+{
+    [ExcludeFromCodeCoverage]
+    public class GetSettingsResponse 
+    {
+        public int ExpiryAfterMonths { get; set; }
+        public int RemovedAfterExpiryMonths { get; set; }
+    }
```
Comment (karen-hanna) [inline src/Shared/SFA.DAS.SharedOuterApi/InnerApi/Responses/RequestApprenticeTraining/GetSettingsResponse.cs:10]: Should this also include ProviderRemovedAfterRequestedMonths ?

File: src/EmployerRequestApprenticeTraining/SFA.DAS.EmployerRequestApprenticeTraining/Application/Queries/GetSettings/GetSettingsResult.cs (hunk 1 above)
Comment (ChrisJWoodcock) [reply src/EmployerRequestApprenticeTraining/SFA.DAS.EmployerRequestApprenticeTraining/Application/Queries/GetSettings/GetSettingsResult.cs:8]: No as this is the employer outer api there is no need for the provider setting

File: src/Shared/SFA.DAS.SharedOuterApi/InnerApi/Responses/RequestApprenticeTraining/GetSettingsResponse.cs (hunk 2 above)
Comment (ChrisJWoodcock) [reply src/Shared/SFA.DAS.SharedOuterApi/InnerApi/Responses/RequestApprenticeTraining/GetSettingsResponse.cs:11]: Yes it should as this is the shared response

Comment (ChrisJWoodcock) [reply src/Shared/SFA.DAS.SharedOuterApi/InnerApi/Responses/RequestApprenticeTraining/GetSettingsResponse.cs:11]: Done

**Review by karen-hanna — APPROVED** [review]

**Comment by ChrisJWoodcock** [comment]
> ## [Quality Gate Failed](https://sonarcloud.io/dashboard?id=SkillsFundingAgency_das-apim-endpoints_ProviderRequestApprenticeTraining&pullRequest=1616) **Quality Gate failed for 'das-apim-endpoints-ProviderRequestApprenticeTraining'**
> Failed conditions  [29.9% Coverage on New Code](https://sonarcloud.io/component_measures?id=SkillsFundingAgency_das-apim-endpoints_ProviderRequestApprenticeTraining&pullRequest=1616&metric=new_coverage&view=list) (required ≥ 80%)
> 
> [See analysis details on SonarCloud](https://sonarcloud.io/dashboard?id=SkillsFundingAgency_das-apim-endpoints_ProviderRequestApprenticeTraining&pullRequest=1616)

This is caused by shared code which is added for the Employer outer api being uncovered in the Provider outer api because it isn't used but is in the shared code library so causes these coverage issues - will have to ignore that

**Review by karen-hanna — APPROVED** [review]

**Pull request merged**